topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/faqs/remaining_human_sequences.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Isn't it awkward to find so many humans sequences there, since we filter for them before?
+box_type: question
+layout: faq
+contributors: [EngyNasr]
+---
+
+We see a lot that Kraken tends to assign many reads to human, despite they do not map to human genome. Due to resemblance between organisms and the limited species coverage of k2 DBs sometimes does happen that reads corresponding to higher organisms get mapped to humans. It was a very severe problem for the standard DB, because yeast genes were mis-assigned to human.
```

Comment:
```suggestion
We see a lot that Kraken tends to assign many reads to human, despite they do not map to human genome. Due to resemblance between organisms and the limited species coverage of Kraken databases sometimes does happen that reads corresponding to higher organisms get mapped to humans. It was a very severe problem for the standard databases, because yeast genes were mis-assigned to human.
```